wiremock/mappings/inapp-v2.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This file is only for vast request. Wasn't there mocked response for other ad units before ? Maybe should you rename this accordingly and makes the preconditions more precise ?

Reply:
Updated with a precondition on query to only respond to rewarded ones.